test/e2e/test_tools.py

```diff
@@ -128,117 +129,132 @@ def test_create_project(self) -> None:
 
 
     def test_get_ontology(self) -> None:
```

Comment:
this method does way more than testing. Could you extract the json modification into a/some helper method(s)?